### ISSUE TITLE
Use own Icons

### ISSUE THREAD

**Issue by Speeedy0815** [FlowFuse/node-red-dashboard#1412]
### Description

Hello, 

I want to use my own icons on buttons. Is there a (easy) way to put them on the server in an special directorty and use them as icons for the buttons?

Usecase: For Homeautomation there are a lot of icons missing. (for example 20%-opened-blinds)

Because it is a special usecase and you can not provide tousends of icons I want to have a possibility to use my own.

Thank you.

### Have you provided an initial effort estimate for this issue?

I am no FlowFuse team member

**Comment by joepavitt** [comment]
We don't currently, but support the [Material Design Icon library](https://pictogrammers.com/library/mdi/), which has nearly 7.5k icons included by default, including many blind-based icons:

<img width="1470" alt="Screenshot 2024-10-18 at 17 11 54" src="https://github.com/user-attachments/assets/c8132d30-dbf6-4b2c-9542-4c59df6e7112">

**Comment by Speeedy0815** [comment]
Yes, I know. That’s why I marked it as a feature request, not a bug.
I’ll take a look at the new library and hope it has enough to meet my needs. However, I still think it would be a useful feature to have.

**Comment by joepavitt** [comment]
>  However, I still think it would be a useful feature to have.

Agreed, thanks for raising it

**Comment by joepavitt** [comment]
Thanks @TheeeEnding - for the App Icon, you should be able to point to a local file and it will render
